share/toolchain-nxdk.cmake

```diff
@@ -1,4 +1,8 @@
-set(NXDK_DIR $ENV{NXDK_DIR})
+if(DEFINED ENV{NXDK_DIR})
+    set(NXDK_DIR $ENV{NXDK_DIR})
+else()
+    message(FATAL_ERROR "The environment variable NXDK_DIR needs to be defined.")
+endif()
```

Comment:
[Please check history](https://github.com/XboxDev/nxdk/commits/master) for commit-style.

Reply:
I see, I redid the commit for both PRs.